### DISCUSSION TITLE
Possible to remove fief branding?

### DISCUSSION THREAD

**Discussion by ukena** [fief-dev/fief#291] [Q&A]
Is it possible to remove the "Powered and secured by fief" branding?

**Comment by fief-bailiff** [comment]
Hail, @ukena 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!

**Reply by frankie567** [reply]
Hey! Not currently, but that's something we plan to do when launching paid offers.
